lib/error-debug.js

```diff
@@ -40,8 +40,8 @@ const styles = {
   },
 
   message: {
-    fontFamily: '"SF Mono", "Roboto Mono", "Fira Mono", menlo-regular, monospace',
-    fontSize: '10px',
+    fontFamily: '"SF Mono", "Roboto Mono", "Fira Mono", consolas, menlo-regular, monospace',
+    fontSize: '14px',
```

Comment:
Just tried on my retina macbook pro. I'd rather have it at `13px` (yes I know I'm asking only 1 pixel decrease :S). And then add `15px` to the heading. Since the heading is smaller than the message in your PR. Other than that 👍